test/OneShot.purs

```diff
@@ -0,0 +1,194 @@
+module Test.OneShot
```

Comment:
We can just move the tests and contracts to our existing `Test.Plutip` rather than having a dedicated module